vulnerabilities/package_managers.py

```diff
@@ -217,12 +265,15 @@ class MavenVersionAPI(VersionAPI):
     async def load_api(self, pkg_set):
         async with client_session() as session:
             await asyncio.gather(
-                *[self.fetch(pkg, session) for pkg in pkg_set if pkg not in self.cache]
+                *[
+                    self.fetch(pkg, session)
+                    for pkg in pkg_set
+                    if pkg not in self.cache and "camel" not in pkg
```

Comment:
Why special case "camel"? I can accept that there could be problem, but this should NOT be handled this way. It should be super explicit and explained thoroughly.